### ISSUE TITLE
Improve color blind accessibility

### ISSUE THREAD

**Issue by djfpaagman** [joeldrapper/quickdraw#32]
I have light deuteranopia, which makes it hard to distinguish red and green, so the green and red dots in the output are very hard to parse for me. It would be nice to make it a bit more accessible in any case, we could:

* add some kind of alternative color mode that you can configure
* change the default colors
* not use colored circles (boring!) and use something like `.` and `F` like other testing frameworks
* make the above optionally configurable as well

What do you think?

**Comment by joeldrapper** [comment]
Absolutely, yes. It’s not documented, but you can actually configure the success and failure symbols to be any string you want.  I think we should make the colours configurable and also have better defaults. Maybe 🟢 and 🚫? Or ✅ and ❌.

https://github.com/joeldrapper/quickdraw/blob/37ad75d439618e12509368ec4b507a9225e09baa/lib/quickdraw/configuration.rb#L11-L12
